test/validation/api/pktio/pktio.c

```diff
@@ -3468,6 +3468,109 @@ static void pktio_test_recv_maxlen_set(void)
 	packet_len = PKT_LEN_NORMAL;
 }
 
+static int pktio_check_pktout_aging_tmo(void)
+{
+	odp_pktio_param_t pktio_param;
+	odp_pktio_capability_t capa;
+	odp_pktio_t pktio;
+	int idx = 0;
+	int ret;
+
+	odp_pktio_param_init(&pktio_param);
+	pktio_param.in_mode = ODP_PKTIN_MODE_DIRECT;
+	pktio_param.out_mode = ODP_PKTOUT_MODE_DIRECT;
+
+	pktio = odp_pktio_open(iface_name[idx], pool[idx], &pktio_param);
+	if (pktio == ODP_PKTIO_INVALID)
+		return ODP_TEST_INACTIVE;
+
+	ret = odp_pktio_capability(pktio, &capa);
+	(void)odp_pktio_close(pktio);
+
+	if (ret < 0 || !capa.config.pktout.bit.aging_ena)
```

Comment:
This should check max_tx_aging_tmo_ns capability instead. That's documented to be 0 when not supported. We should remove .config from capa at some point.